index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Andrii Kravchenko">
+    <meta name="description" content="USING HASKELL WITH 5- TO 7-YEAR-OLDS - how to teach children to code on Haskell? Read the article..." >
+    <link rel="icon" href="../myfirstproject/img/Go_gopher_favicon.svg" type="image/x-icon"/>
+    <meta charset="UTF-8">
+    <title>HASKELL</title>
+</head>
+<body>

```

Comment:
I see that used much `span` tag, but why didn't you use `strong`, `small`, `em` etc?